src/Microsoft.AspNetCore.App/build/netcoreapp2.1/Microsoft.AspNetCore.App.targets

```diff
@@ -0,0 +1,7 @@
+<Project InitialTargets="EnsureTFMCompatibility">
```

Comment:
Missed creating this check when I separated the shared framework into two.

Reply:
Out of curiosity, is there a reason we do this as opposed to wiring up to a specific target such as `BeforeBuild`?

Reply:
Not particularly, but is there anything wrong with this approach?

Reply:
There aren't very strong reasons, but the one thing I can think of is that there really is no way for a user to turn it off (short of deleting the target) by manipulating the csproj. Just a thought.

Reply:
Would be good to see how this behaves in VS and VS Code. InitialTargets will run in the design-time builds that power the UI in VS and Omnisharp in VSCode